### PR TITLE
[Docs] Add risk mitigation links

### PR DESCRIPTION
**Pull request by nadijagraca**
## Description
Added links in the models section of the docs to provide guidance on risk mitigation, advising users to review  before using the models.

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

  - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
  - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorized to submit this contribution on behalf of the original creator(s) or their licensees.
  - I certify that the use of this contribution as authorized by the Apache 2.0 license does not violate the intellectual property rights of anyone else.
  - I have not referenced individuals, products or companies in any commits, directly or indirectly.
  - I have not added data or restricted code in any commits, directly or indirectly.

### CODE REVIEW

**Review by petar-qb — APPROVED** [review]
This looks good to me!

File: vizro-ai/changelog.d/20241112_202616_nadija_ratkusic_graca_add_risk_mitigation_links.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<!--
```
Comment (stichbury) [inline vizro-ai/changelog.d/20241112_202616_nadija_ratkusic_graca_add_risk_mitigation_links.md:1]: You shouldn't need this now -- no changelogs for just-docs changes.

File: vizro-ai/docs/pages/user-guides/customize-vizro-ai.md [hunk 2]
```diff
@@ -24,7 +24,8 @@ vizro_ai = VizroAI(model="<chosen model>")
     | API key           | `OPENAI_API_KEY`                      |
     | Base API URL      | `OPENAI_API_BASE`                     |
 
-    To use OpenAI with Vizro-AI, you must have an account with paid-for credits available. None of the free accounts will suffice. You can check [all available OpenAI models including pricing on their website](https://platform.openai.com/docs/models). This will also explain which version the below string acronyms currently point to.
+    To use OpenAI with Vizro-AI, you must have an account with paid-for credits available. None of the free accounts will suffice. You can check [all available OpenAI models including pricing on their website](https://platform.openai.com/docs/models). This will also explain which version the below string acronyms currently point to. Before usage, please review OpenAI's guidelines on risk mitigation to understand potential model limitations and best practices.
```
Comment (stichbury) [inline vizro-ai/docs/pages/user-guides/customize-vizro-ai.md:27]:
```suggestion
    To use OpenAI with Vizro-AI, you must have an account with paid-for credits available. None of the free accounts will suffice. [Check the OpenAI models and pricing on their website](https://platform.openai.com/docs/models). Before using a model, please review OpenAI's guidelines on risk mitigation to understand potential model limitations and best practices.
```

File: vizro-ai/docs/pages/user-guides/customize-vizro-ai.md [hunk 3]
```diff
@@ -24,7 +24,8 @@ vizro_ai = VizroAI(model="<chosen model>")
...
+    More details on responsible usage can be found on this [link](https://platform.openai.com/docs/models).
```
Comment (stichbury) [inline vizro-ai/docs/pages/user-guides/customize-vizro-ai.md:28]:
```suggestion
    [See the OpenAI site for more details on responsible usage](https://platform.openai.com/docs/models).
```

File: vizro-ai/docs/pages/user-guides/customize-vizro-ai.md [hunk 4]
```diff
@@ -62,6 +64,7 @@ vizro_ai = VizroAI(model="<chosen model>")
     | Base API URL      | `MISTRAL_BASE_URL`                        |
 
     To use Mistral with Vizro-AI, you can either use their API, which comes with [an associated cost](https://mistral.ai/technology/#pricing), or you could use their models for free under the Apache 2.0 license. In that case you need to setup the model API yourself. You can check [all available Mistral models including pricing on their website](https://docs.mistral.ai/getting-started/models/models_overview). This will also explain which version the below string acronyms currently point to.
+    Before usage, please review Mistral guidelines on risk mitigation to understand potential model limitations and best practices. More details on responsible usage can be found on this [link](https://docs.mistral.ai/capabilities/guardrailing/#guardrailing-results-with-mistral-safety-prompt).
```
Comment (stichbury) [inline vizro-ai/docs/pages/user-guides/customize-vizro-ai.md:67]:
```suggestion
    Before using a model, please review Mistral guidelines on risk mitigation to understand potential model limitations and best practices. [See the Mistral site for more details on responsible usage](https://docs.mistral.ai/capabilities/guardrailing/#guardrailing-results-with-mistral-safety-prompt).
```

**Review by stichbury — APPROVED** [review]
LGTM, just a few suggested changes.

File: vizro-ai/docs/pages/user-guides/customize-vizro-ai.md (hunk 3 above)
Comment (maxschulz-COL) [reply vizro-ai/docs/pages/user-guides/customize-vizro-ai.md:28]: Should we not link to this: https://platform.openai.com/docs/guides/safety-best-practices. I have only browsed this, but it seems more relevant?

File: vizro-ai/docs/pages/user-guides/customize-vizro-ai.md [hunk 5]
```diff
@@ -39,7 +40,8 @@ vizro_ai = VizroAI(model="<chosen model>")
     | API key           | `ANTHROPIC_API_KEY`                       |
     | Base API URL      | `ANTHROPIC_API_URL`,`ANTHROPIC_BASE_URL`  |
 
-    To use Anthropic with Vizro-AI, you must have an account with paid-for credits available. None of the free accounts will suffice. You can check [all available Anthropic models including pricing on their website](https://docs.anthropic.com/en/docs/about-claude/models).
+    To use Anthropic with Vizro-AI, you must have an account with paid-for credits available. None of the free accounts will suffice. You can check [all available Anthropic models including pricing on their website](https://docs.anthropic.com/en/docs/about-claude/models). Before usage, please review Anthropic guidelines on risk mitigation to understand potential model limitations and best practices.
+    More details on responsible usage can be found on this [link](https://trust.anthropic.com/).
```
[the remark this answers is not in the document]
Comment (maxschulz-COL) [reply vizro-ai/docs/pages/user-guides/customize-vizro-ai.md:44]: WHat about this link: https://support.anthropic.com/en/collections/4078535-trust-safety

File: vizro-ai/docs/pages/user-guides/customize-vizro-ai.md (hunk 4 above)
Comment (maxschulz-COL) [reply vizro-ai/docs/pages/user-guides/customize-vizro-ai.md:67]: The problem with this link is that I am not sure we have that enabled atm? The other two vendors have more general advice, but here it is really about the API call. Should we enable this? And is there potentially more general info?

File: vizro-ai/docs/pages/user-guides/customize-vizro-ai.md [hunk 6]
```diff
@@ -62,6 +64,8 @@ vizro_ai = VizroAI(model="<chosen model>")
     | Base API URL      | `MISTRAL_BASE_URL`                        |
 
     To use Mistral with Vizro-AI, you can either use their API, which comes with [an associated cost](https://mistral.ai/technology/#pricing), or you could use their models for free under the Apache 2.0 license. In that case you need to setup the model API yourself. You can check [all available Mistral models including pricing on their website](https://docs.mistral.ai/getting-started/models/models_overview). This will also explain which version the below string acronyms currently point to.
+    Before usage, please review Mistral guidelines on risk mitigation to understand potential model limitations and best practices.
+    [See the Mistral site for more details on responsible usage](https://help.mistral.ai/en/collections/280418-legal-security-and-data-privacy/).
```
Comment (maxschulz-COL) [inline vizro-ai/docs/pages/user-guides/customize-vizro-ai.md:68]: Not sure this link applies, do you think this one might be better? https://help.mistral.ai/en/collections/272960-le-chat

**Pull request merged**